src/js/site.js

```diff
@@ -86,5 +90,75 @@ jquery.fn.sui_module = function () {
   });
 };
 
+/**
+ * Plausible tracking module
+ *
+ * This reuses jQuery to provide explicit tracking of events at Plausible. To
+ * use events, add the ``data-analytics`` attribute to an element. In most
+ * cases, this should be a link element, however in the case of other UI
+ * components, it may be a ``<div>`` or ``<button>``:
+ *
+ *     <button class="ui button" data-analytics="some-event-id">Something</button>
+ *
+ * In the case of a link with a ``href`` attribute, the link will continue
+ * redirecting after the callback from Plausible fires off.
+ */
+jquery.fn.plausible = function () {
+  let plausible_settings = { domain: "about.readthedocs.org" };
+  if (DEBUG_MODE) {
+    plausible_settings.trackLocalhost = true;
+  }
+  const { trackEvent } = Plausible(plausible_settings);
+  return this.each((index, elem) => {
+    elem.addEventListener("click", on_click_event);
+    elem.addEventListener("auxclick", on_click_event);
+
+    function on_click_event(event) {
+      const event_names = elem.getAttribute("data-analytics").split(/,(.+)/);
+      const is_link =
+        elem.tagName != undefined && elem.tagName.toLowerCase() == "a";
+      const is_middle_click = event.type == "auxclick" && event.which == 2;
+      const is_click = event.type == "click";
+      const is_link_click =
+        is_link &&
+        is_click &&
+        !elem.target &&
+        !(event.ctrlKey || event.metaKey || event.shiftKey);
+
+      if (is_middle_click || is_click) {
+        function redirect_link() {
+          if (is_link_click && elem.href && elem.href != "#") {
```

Comment:
I had a random thought, and this logic might not be correct for anchor links. Let me double check this really quick.

Reply:
Nevermind, works as expected! My concern was `href="#foo"` would not have the default event propagation, but it does indeed work.